test/GsnTest.test.ts

```diff
@@ -0,0 +1,17 @@
+import GsnTest from '../src/relayclient/GsnTest'
```

Comment:
why do you use the "fixture" as instance, and not class ?
I thinkg that adding gsnTest = new GsnTest() makes it clearer (and also let you initialize different fixtures)

Reply:
This is an experiment in node-controlled singletons. I want to allow only one instance to exist simultaneously.

Reply:
There is no valid reason to hold two